prac_06/guitar.py

```diff
@@ -0,0 +1,14 @@
+class Guitar:
+    def __init__(self, name="", year=0, cost=0):
+        self.name = name
+        self.year = year
+        self.cost = cost
+
+    def get_age(self, current_year):
```

Comment:
get_age(self, current_year): This method calculates the age of the guitar by subtracting the year it was made (self.year) from the current year provided as an argument